### ISSUE TITLE
Additional Prerequisite to call out: Docker installed.

### ISSUE THREAD

**Issue by jgardsentry** [3shape/docker-ci#119]
It may see obvious, but I'm hoping to find a tool to allow me to pull, tag, and push images without having the docker daemon running locally.  I found (understandably) that this module requires docker to be installed and running.  May want to add that to README.

**Comment by zionyx** [comment]
Yes, you are right. This PS module is merely a wrapper for docker commands and still relies entirely on docker daemon to function. I'll update README to make this requirement clear.
